script.js

```diff
@@ -0,0 +1,64 @@
+$(document).ready(function(){
+    const movieGluAPI = '5zwPt8bzCu7r7rifnKoCd827XemsTLcO6rk51vTy';
```

Comment:
Is this the API key for movieGlu? If so we should have the variable name represent that it is the api key

Reply:
I am just going to remove all the other API info until we decide on the second one we will be using.